scripts/publish-android.sh

```diff
@@ -42,7 +42,7 @@ publish_plugin () {
                 if grep --quiet "BUILD SUCCESSFUL" $LOG_OUTPUT; then
                     printf %"s\n\n" "Success: $PLUGIN_NAME published to MavenCentral."
                 else
-                    printf %"s\n\n" "Error publishing $PLUGIN_NAME, check $LOG_OUTPUT for more info! Manual publication review may be necessary at the Sonatype Repository Manager https://s01.oss.sonatype.org/"
+                    printf %"s\n" "Error publishing, check $LOG_OUTPUT for more info! Manually review and release from the Central Portal may be necessary https://central.sonatype.com/publishing/deployments/"
```

Comment:
Can we adjust the beginning of the print statement here to:

```
printf %"s\n\n" "Error publishing $PLUGIN_NAME, check $LOG_OUTPUT for more info!
```

Just so that the extra line break and name of the plugin are retained in the new error message.